drivers/amazonec2/amazonec2.go

```diff
@@ -903,15 +889,13 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) []*
 	hasDockerPort := false
 	hasSwarmPort := false
 	for _, p := range group.IpPermissions {
-		if p.FromPort != nil {
```

Comment:
yeah ,   I don’t think thats my latest commit,   I had some problems
rebasing and I think that resulted in some weird stuff,  Let me get a new
PR up so you can take a look at a clean diff.

On Fri, Jan 15, 2016 at 2:12 PM, Nathan LeClaire <notifications@github.com>
wrote:

> In drivers/amazonec2/amazonec2.go
> <https://github.com/docker/machine/pull/1956#discussion_r49892543>:
>
> > @@ -903,15 +889,13 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) []*
> >  	hasDockerPort := false
> >  	hasSwarmPort := false
> >  	for _, p := range group.IpPermissions {
> > -		if p.FromPort != nil {
>
> This p.FromPort != nil line should not be removed. It was introduced in
> f17cce330c5b4b7a0b04d3c931d3a96cdff53476 to prevent a panic from occuring.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/machine/pull/1956/files#r49892543>.
>